### PR TITLE
Fixes to release.js

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
r? ... we should probably just create a needs review label.

File: tools/release.js [hunk 1]
```diff
@@ -175,18 +189,34 @@ function getBranchState() {
   });
 }
 
+function getCurrentBranch(){
+  return async.task(function*(){
+     const branch = yield git(`rev-parse --abbrev-ref HEAD`);
```
Comment (halindrome) [inline tools/release.js]: Clever

**Comment by halindrome** [comment]
LGTM

File: tools/release.js (hunk 1 above)
Comment (marcoscaceres) [inline tools/release.js]: Credit to stack overflow for that one:)

**Pull request merged**